hack/e2e.sh

```diff
@@ -49,12 +49,27 @@ on_exit() {
 
   # Cleanup VSPHERE_PASSWORD from temporary artifacts directory.
   if [[ "${ORIGINAL_ARTIFACTS}" != "" ]]; then
+    # Delete non-text files from artifacts directory to not leak files accidentially
```

Comment:
Can we make a bit surer that this is only run over ARTIFACTS? :) Just because it's rm.. :D

Reply:
You're too quick 😂